openshift-hack/images/os/Dockerfile

```diff
@@ -16,8 +16,9 @@ RUN set -x && yum install -y ostree rpm-ostree yum-utils selinux-policy-targeted
     mkdir -p /tmp/tmprootfs/etc && \
     ostree --repo=/srv/repo checkout -U $commit --subpath /usr/etc/selinux /tmp/tmprootfs/etc/selinux && \
     ostree --repo=/srv/repo commit --parent=$commit --tree=ref=$commit --tree=dir=. \
-        --selinux-policy /tmp/tmprootfs \
-        -s "origin-ci-dev overlay RPMs" --branch=origin-ci-dev
+      --selinux-policy /tmp/tmprootfs \
+      -s "origin-ci-dev overlay RPMs" --branch=origin-ci-dev && \
+    cp -r /extensions /srv/repo/
 
 FROM scratch
 COPY --from=build /srv/ /srv/
```

Comment:
I think we also need to do `RUN mv /srv/extensions /`.  Or probably better, just don't do the `cp` above and just `COPY --from=build /extensions/ /extensions/` or so?